src/community-guides/resources-to-get-started-in-cybersecurity.md

```diff
@@ -0,0 +1,89 @@
+# Resources To Get Started in Cybersecurity
+
+## What is cybersecurity ?
+
+Cybersecurity is protection of computer systems or networks from malicious actors that may result in theft of data or disruption to any services these systems provide. It has many fields such as web hacking , mobile hacking , reverse engineering etc. To get started, you have to pick the field which interests you. It can be web hacking if you like to tinker around web applications to see how they behave or maybe reverse engineering , if you like to see how the applications works.
+
+
+### Basics
+
+To get, it is recommended to have basic understanding of computer networking as to hack something you have to know how it communicates.
+
+[Network Fundamentals](https://www.youtube.com/watch?v=9uebakqWlB0)
+
+[TCP IP Model](https://www.youtube.com/watch?v=e5DEVa9eSN0)
+
+[How ssl works](https://howhttps.works/)
+
+[Basic of HTTP](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview)
+
+If you want a university like course with deeper understanding of basics you can follow
+
+[PWN College](https://pwn.college/intro-to-cybersecurity/)
```

Comment:
Lets use a list, since its a list and is much easier to read :)

```suggestion
- [Network Fundamentals](https://www.youtube.com/watch?v=9uebakqWlB0)
- [TCP IP Model](https://www.youtube.com/watch?v=e5DEVa9eSN0)
- [How ssl works](https://howhttps.works/)
- [Basic of HTTP](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview)
- If you want a university like course with deeper understanding of basics you can follow [PWN College](https://pwn.college/intro-to-cybersecurity/)
```